public/app/features/alerting/unified/mocks.ts

```diff
@@ -438,6 +451,37 @@ export const someRulerRules: RulerRulesConfigDTO = {
   namespace2: [mockRulerRuleGroup({ name: 'group3', rules: [mockRulerAlertingRule({ alert: 'alert3' })] })],
 };
 
+export const someCloudRulerRules: RulerRulesConfigDTO = {
```

Comment:
nit: These two methods seem to be highly coupled to the `EditRuleGroupModal` test. I'd keep them in the test file